fixitfriday.ui/src/components/Survey/SurveyVisivilityAdministration/SurveyVisibilityAdministration.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useCallback, FC } from 'react';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+import PageHeader from './pageHeader';
+import { SurveyQuestionType } from '../types/SurveyQuestionType';
+
+import SectionSurveyResultsData from '../mockData/SectionSurveyResultsData';
+import SurveyQuestions from './SurveyQuestions';
+import { SurveyDefinitionType } from '../types/SurveyDefinitionType';
+
+const headerData: Array<SurveyDefinitionType> = SectionSurveyResultsData.surveydefinition;
+
+const SurveyVisibilityAdministration: FC = () => {
+  const [surveyKey, setSectionKey] = useState<string>('');
+  const [SurveysData, setSurveyQuestionsData] = useState<Array<SurveyQuestionType>>([]);
+
+  const [surveyDisabled, setSurveyAvailability] = useState<boolean>(false);
```

Comment:
```suggestion
  const [surveyAvailability, setSurveyAvailability] = useState<boolean>(false);
```
It is better to try to always match the setter with the state name.